base/v0_1/translate_test.go

```diff
@@ -18,21 +18,49 @@ import (
 	"reflect"
 	"testing"
 
+	"github.com/coreos/fcct/translate"
+
 	"github.com/coreos/ignition/v2/config/util"
 	"github.com/coreos/ignition/v2/config/v3_0/types"
+	"github.com/coreos/vcontext/path"
 )
 
 // Most of this is covered by the Ignition translator generic tests, so just test the custom bits
 
+// verifyTranslations ensures all the translations are identity, unless they match a listed one
+// it returns the offending translation if there is one
+func verifyTranslations(set translate.TranslationSet, exceptions ...translate.Translation) *translate.Translation {
+	exceptionSet := translate.TranslationSet{
+		FromTag: set.FromTag,
+		ToTag:   set.ToTag,
+		Set:     map[string]translate.Translation{},
```

Comment:
I see `map[string]Translation` used a number of times in the code. Consider making it it's own type.

Reply:
That's basically what TranslationSet is. In practice users don't initialize TranslationSets themselves much since Translate() does that. It's basically just tests where we need to have a bunch a literals to compare against, so I'm inclined to leave it as is.